cobbler.spec

```diff
@@ -37,6 +37,8 @@
 %define grub2_x64_efi_pkg grub2-x86_64-efi
 %define grub2_ia32_efi_pkg grub2-i386-efi
 %define system_release_pkg distribution-release
+%undefine python_enable_dependency_generator
+%undefine python_disable_dependency_generator
```

Comment:
Wrap these `undefine` statements in `0%{?suse_version} && 0%{?suse_version} < 1550` conditional, perhaps?